charts/knative-serving-istio/templates/istio/istio-namespace.yaml

```diff
@@ -0,0 +1,6 @@
+# {{- if .Values.base.enabled }}
```

Comment:
why keep the file? 

or else, Shall we point to the description you added in `release-manifests.yaml` as a comment in the start of this file?

Reply:
same as above